seqr/utils/elasticsearch/es_search.py

```diff
@@ -434,7 +440,7 @@ def _process_single_search_response(self, parsed_response, page=1, num_results=1
         return variant_results[:num_results]
 
     def _execute_multi_search(self, **kwargs):
-        indices = self._index_searches.keys() or self._indices
+        indices = sorted(list(self._index_searches.keys()) or self._indices, reverse = True)
```

Comment:
In Python 3, the order of `list(self._index_searches.keys())` is not fixed. Therefore, `sorted()` is added.

Reply:
so this is an okay change to make, but its confusing to me because in python2 dicts are not ordered by default, but in python 3 there are. So I would expect the opposite

Reply:
In Python 2, `dict.keys()` produces a sequence order (increasing) while starting from Python 3.7 it preserves the order of adding the keys. Below are the examples:
Python 2:
```
>>> d = {'a': 0, 'b': 1, 'c': 2}
>>> e = {'b':0, 'a':1, 'c': 2}
>>> d.keys()
['a', 'c', 'b']
>>> e.keys()
['a', 'c', 'b']
>>> 
```
Python 3.7 or higher:
```
>>> d = {'a': 0, 'b': 1, 'c': 2}
>>> e = {'b':0, 'a':1, 'c': 2}
>>> d.keys()
dict_keys(['a', 'b', 'c'])
>>> e.keys()
dict_keys(['b', 'a', 'c'])
>>> 
```